src/functions/module_filter.ts

```diff
@@ -221,12 +221,20 @@ export const SORTN: AddFunctionDescription = {
   compute: function (
```

Comment:
should remove the isExported (or set to False)

Reply:
Up

Reply:
to fix in previous versions